digicampipe/calib/camera/r1.py

```diff
@@ -26,7 +26,7 @@ def calibrate_to_r1(event_stream, dark_baseline):
                 r1_camera.nsb = calib.compute_nsb_rate(standard_deviation, 'std')
             else:
                 # compute NSB and Gain drop from baseline shift
-                r0_camera.dark_baseline = dark_baseline['baseline']
+                r0_camera.dark_baseline = dark_baseline
```

Comment:
This block is the else statement of : 

```python
if dark baseline is None:
```
with the change in the code now it is not doing the same. Previously it was checking if the argument passed in the func. `dark_baseline` was `None`. Now it checks if `dark_baseline = dark_baseline['baseline']` is None.

Reply:
Oh .. right.